.trellis/scripts/common/developer.py

```
@@ -0,0 +1,190 @@
+    if not name:
+        print("Error: developer name is required", file=sys.stderr)
+        return False
+
+    if repo_root is None:
+        repo_root = get_repo_root()
+
+    dev_file = repo_root / DIR_WORKFLOW / FILE_DEVELOPER
+    workspace_dir = repo_root / DIR_WORKFLOW / DIR_WORKSPACE / name
+
+    # Create .developer file
+    initialized_at = datetime.now().isoformat()
```

Ruff (0.15.18)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)